machine.go

```diff
@@ -907,8 +911,15 @@ func (m *Machine) sendCtrlAltDel(ctx context.Context) error {
 	return err
 }
 
-func (m *Machine) setMmdsConfig(ctx context.Context, address net.IP, ifaces NetworkInterfaces) error {
+func (m *Machine) setMmdsConfig(ctx context.Context, address net.IP, ifaces NetworkInterfaces, version string) error {
 	var mmdsCfg models.MmdsConfig
+	// MMDS config supports v1 and v2, v1 is going to be deprecated.
+	// Default to the version 1 if no version is specified
+	if version == "V1" || version == "V2" {
```

Comment:
This seems like it will be a source of confusion. Passing "v2" will silently fall back to MMDSv1. 

Rather than exposing the version as a string in the go API, I think we should create a new type with constants for v1 and v2 to make it easier to get this right:
```
type MMDSVersion string

const (
    MMDSv1 = MMDSVersion("V1")
    MMDSv2 = MMDSVersion("V2")
)
```

TBH, I wish LogLevel worked like this too, but that ship has sailed.

Reply:
Done !